english/light-fire/burn-brightly/burn-brightly.md

```diff
@@ -0,0 +1,4 @@
+You enjoy the sweet hot marshmallows for some seconds and then the pain overcomes your taste-buds. You start shrieking horribly and you realise it's too late. Like the marshmallows you too are BURNING BRIGHTLY!! 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.